middleware/src/rpcmessages/errorcodes.go

```diff
@@ -139,6 +139,16 @@ const (
 	ErrorSetHostnameInvalidValue ErrorCode = "SET_HOSTNAME_INVALID_VALUE"
 )
 
+const (
+
+	/* bbb-systemctl.sh start-bitcoin-services
+	---------------------------------------*/
+
+	// ErrorSystemdServiceStartFailed is thrown when the systemd services can not
```

Comment:
"when a systemd service cannot be started"? Could be used in other context as well and indicates usually a single failed attempt.